app/pages/examples.njk

```diff
@@ -33,9 +33,9 @@
     <li><a href="../components/card/card-with-image.html">Card - Card with an image</a></li>
     <li><a href="../components/card/card-group.html">Card - Card group</a></li>
     <li><a href="../components/card/feature-card.html">Card - Feature card</a></li>
-    <li><a href="../components/care-card/care-card-non-urgent.html">Care card non-urgent (blue)</a></li>
-    <li><a href="../components/care-card/care-card-urgent.html">Care card urgent (red)</a></li>
```

Comment:
Won't this need to be ".../card/care-card-urgent.html" with the files changes in this PR?